modules/actions/rapid_accept_feature.js

```diff
@@ -65,6 +65,11 @@ function removeMetadata(entity) {
     delete entity.__datasetid__;
     delete entity.tags.conn;
     delete entity.tags.dupe;
+
+    //These three new values are specific to the mapillary objects
```

Comment:
Before rapid features are added to the map, the data objects are decorated with a bunch of different things to help rapid make sense of them. This method just removes that data once it's part of the OSM layer/map.